shared/js/npm-debug.log

```diff
@@ -0,0 +1,44 @@
+0 info it worked if it ends with ok
+1 verbose cli [ 'node',
+1 verbose cli   '/Users/andrew/.nvm/versions/node/v0.12.4/bin/npm',
+1 verbose cli   'run',
+1 verbose cli   'gulp' ]
+2 info using npm@3.4.0
+3 info using node@v0.12.4
+4 verbose run-script [ 'pregulp', 'gulp', 'postgulp' ]
+5 info lifecycle @cdo/shared@0.0.1~pregulp: @cdo/shared@0.0.1
+6 silly lifecycle @cdo/shared@0.0.1~pregulp: no script for pregulp, continuing
+7 info lifecycle @cdo/shared@0.0.1~gulp: @cdo/shared@0.0.1
+8 verbose lifecycle @cdo/shared@0.0.1~gulp: unsafe-perm in lifecycle true
+9 verbose lifecycle @cdo/shared@0.0.1~gulp: PATH: /Users/andrew/.nvm/versions/node/v0.12.4/lib/node_modules/npm/bin/node-gyp-bin:/Users/andrew/Code/code-dot-org/shared/js/node_modules/.bin:/Users/andrew/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/bin:/Users/andrew/.rbenv/versions/2.2.3/bin:/usr/local/Cellar/rbenv/0.4.0/libexec:/Users/andrew/.rbenv/plugins/ruby-build/bin:/Users/andrew/.nvm/versions/node/v0.12.4/bin:/Users/andrew/.rbenv/shims:/Users/andrew/.rbenv/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin
+10 verbose lifecycle @cdo/shared@0.0.1~gulp: CWD: /Users/andrew/Code/code-dot-org/shared/js
+11 silly lifecycle @cdo/shared@0.0.1~gulp: Args: [ '-c', 'gulp' ]
+12 silly lifecycle @cdo/shared@0.0.1~gulp: Returned: code: 1  signal: null
+13 info lifecycle @cdo/shared@0.0.1~gulp: Failed to exec gulp script
+14 verbose stack Error: @cdo/shared@0.0.1 gulp: `gulp`
+14 verbose stack Exit status 1
+14 verbose stack     at EventEmitter.<anonymous> (/Users/andrew/.nvm/versions/node/v0.12.4/lib/node_modules/npm/lib/utils/lifecycle.js:232:16)
+14 verbose stack     at EventEmitter.emit (events.js:110:17)
+14 verbose stack     at ChildProcess.<anonymous> (/Users/andrew/.nvm/versions/node/v0.12.4/lib/node_modules/npm/lib/utils/spawn.js:24:14)
+14 verbose stack     at ChildProcess.emit (events.js:110:17)
+14 verbose stack     at maybeClose (child_process.js:1015:16)
+14 verbose stack     at Process.ChildProcess._handle.onexit (child_process.js:1087:5)
+15 verbose pkgid @cdo/shared@0.0.1
+16 verbose cwd /Users/andrew/Code/code-dot-org/shared/js
+17 error Darwin 15.0.0
+18 error argv "node" "/Users/andrew/.nvm/versions/node/v0.12.4/bin/npm" "run" "gulp"
+19 error node v0.12.4
+20 error npm  v3.4.0
+21 error code ELIFECYCLE
+22 error @cdo/shared@0.0.1 gulp: `gulp`
+22 error Exit status 1
+23 error Failed at the @cdo/shared@0.0.1 gulp script 'gulp'.
+23 error Make sure you have the latest version of node.js and npm installed.
+23 error If you do, this is most likely a problem with the @cdo/shared package,
+23 error not with npm itself.
+23 error Tell the author that this fails on your system:
+23 error     gulp
+23 error You can get their info via:
+23 error     npm owner ls @cdo/shared
+23 error There is likely additional logging output above.
+24 verbose exit [ 1, true ]
```

Comment:
dont want to commit this file. i'd add npm-debug.log to your global .gitignore file